template_feed/Microsoft.DotNet.Common.ItemTemplates/content/WebConfig/web.config

```diff
@@ -4,13 +4,15 @@
   <!-- To customize the asp.net core module uncomment and edit the following section. 
   For more info see https://go.microsoft.com/fwlink/?linkid=838655 -->
   <!--
-  <system.webServer>
-    <handlers>
-      <remove name="aspNetCore"/>
-      <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified"/>
-    </handlers>
-    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" />
-  </system.webServer>
+  <location path="." inheritInChildApplications="false">
+    <system.webServer>
+      <handlers>
+        <remove name="aspNetCore"/>
+        <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified"/>
```

Comment:
Shouldn't this default to ANCMV2? Is this template used for 2.1, 2.2, etc?

Reply:
I think it's totally fine to default to V2. What's the necessary change there?